app/models/user.rb

```
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'json_web_token'
```

RuboCop (1.86.1)

[convention] Add an empty line after magic comments.

(Layout/EmptyLineAfterMagicComment)